.github/workflows/ci-workflow.yml

```diff
@@ -15,3 +18,16 @@ jobs:
       - name: "Run all tests"
         run: |
           make test
+
+  snyk:
```

Comment:
I'm curious why the Github Action workflow isn't blocking the CI pipeline where there are high severity issues showing up in the [Snyk dashboard](https://app.snyk.io/org/bcda).   Does it only check to make sure that no new vulnerabilities were added by a specific change in a feature branch (basically, ignoring existing ones)?

Reply:
So this will block any issues that are found ad hoc with the cli command. It will be most likely more up to date than the one in the dashboard since that is updated daily

Reply:
@mtrang1263 Same question as in `bcda-app`.  Do you want to scan the `Dockerfiles` in addition to `go.mod`?